package com.education.learningplatform.exception;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDateTime;
import java.util.Map;

@Getter
@AllArgsConstructor
public class ErrorResponse {

    private String code;
    private String message;
    private Map<String, String> fields;
    private LocalDateTime timestamp;
}
